app/flatpak-builtins-build-export.c

```diff
@@ -836,6 +866,9 @@ flatpak_builtin_build_export (int argc, char **argv, GCancellable *cancellable,
   if (!flatpak_repo_collect_sizes (repo, root, &installed_size, &download_size, cancellable, error))
     goto out;
 
+  g_variant_dict_insert_value (&metadata_dict, "ostree.commit.collection-ref",
+                               g_variant_new ("(ss)", (collection_id != NULL) ? collection_id : "", full_branch));
```

Comment:
Shouldn't we inherit the current repo collection id if there is one rather than NULL?